ws: "/home/oskar/smooth_sailing/data/my_workspace/"
max_time_interval: 300

run_workspace_preparation: false
run_navigation: false
run_mapping: false
run_post_processing: true

ext_file: "/home/oskar/smooth_sailing/src/smooth_sailing/cfg/ext_right.yaml"
nav_config: "/home/oskar/smooth_sailing/src/smooth_sailing/cfg/nav.yaml"